### PR TITLE
GRC release changes to move to go1.18

### PR DESCRIPTION
**Pull request by gparvin**
Refs:
 - https://github.com/stolostron/backlog/issues/22476

Signed-off-by: Gus Parvin <gparvin@redhat.com>

### CODE REVIEW

File: core-services/prow/02_config/open-cluster-management-io/ocm-kustomize-generator-plugins/_prowconfig.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ branch-protection:
               required_status_checks:
                 contexts:
                 - Linting (1.16)
-                - Unit Tests (1.16)
+                - Unit Tests (1.18)
```
Comment (dhaiducek) [inline core-services/prow/02_config/open-cluster-management-io/ocm-kustomize-generator-plugins/_prowconfig.yaml:16]: We can probably skip this one for now. It needs a major update to keep up with midstream.

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks, @gparvin!

Would you please rebase this PR? It's showing Framework changes already in the main branch.

Also, it looks like we may have overlooked `open-cluster-management-io/governance-policy-addon-controller`, but I'm wondering if that matters. Did you happen to notice whether the status checks were marked as "Required"?

File: core-services/prow/02_config/open-cluster-management-io/ocm-kustomize-generator-plugins/_prowconfig.yaml (hunk 1 above)
Comment (gparvin) [reply core-services/prow/02_config/open-cluster-management-io/ocm-kustomize-generator-plugins/_prowconfig.yaml]: Done, changed this back

**Review by dhaiducek — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**